pubspec.yaml

```diff
@@ -8,6 +8,6 @@ homepage: https://github.com/dart-lang/comid
 environment:
   sdk: '>=1.7.0 <2.0.0'
 dependencies:
-  browser: any
+  browser: '>=0.10.0+2 <0.11.0'
```

Comment:
This dep should probably move to `dev_dependencies`. That way this package won't force clients of this library to use a particular version of `browser`.